### PR TITLE
LinkedIn App Support Added

### PR DESCRIPTION
**Pull request by abhishekjiitr**
Added LinkedIn App support.

### CODE REVIEW

**Comment by mananwason** [comment]
@abhishekjiitr What exactly are you trying to achieve here. Could you add a description?

**Comment by the-dagger** [comment]
@mananwason He is aiming at launching the LinkedIn App directly (if it is installed) once someone taps its icon in the Speaker Details Activity.

**Comment by abhishekjiitr** [comment]
absolutely correct, @the-dagger explained it right @mananwason  . But travis-ci check failed. Any help here?

**Comment by abhishekjiitr** [comment]
@championswimmer  see #266 .
I have made these changes in a single commit there.

File: app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java [hunk 1]
```diff
@@ -2,18 +2,26 @@
...
 import android.net.Uri;
+import android.util.Log;
 import android.view.View;
 import android.widget.ImageView;
 
 import org.fossasia.openevent.R;
 import org.fossasia.openevent.data.Speaker;
 
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java]: Codacy Issue found: [Avoid unused imports such as 'java.net.MalformedURLException'](https://www.codacy.com/app/triplez/open-event-android/file/2406715500/issues/source?bid=3233850&fileBranchId=3233904#l13)

File: app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java [hunk 2]
```diff
@@ -2,18 +2,26 @@
...
+import java.net.URL;
+import java.net.URLConnection;
+import java.net.URLDecoder;
+import java.net.URLEncoder;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java]: Codacy Issue found: [Avoid unused imports such as 'java.net.URLEncoder'](https://www.codacy.com/app/triplez/open-event-android/file/2406715500/issues/source?bid=3233850&fileBranchId=3233904#l17)

File: app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java [hunk 3]
```diff
@@ -2,18 +2,26 @@
...
+import android.util.Log;
 import android.view.View;
 import android.widget.ImageView;
 
 import org.fossasia.openevent.R;
 import org.fossasia.openevent.data.Speaker;
 
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java]: Codacy Issue found: [Avoid unused imports such as 'java.net.URL'](https://www.codacy.com/app/triplez/open-event-android/file/2406715500/issues/source?bid=3233850&fileBranchId=3233904#l14)

File: app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java [hunk 4]
```diff
@@ -2,18 +2,26 @@
...
 import android.content.Intent;
 import android.net.Uri;
+import android.util.Log;
 import android.view.View;
 import android.widget.ImageView;
 
 import org.fossasia.openevent.R;
 import org.fossasia.openevent.data.Speaker;
 
+import java.net.HttpURLConnection;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java]: Codacy Issue found: [Avoid unused imports such as 'java.net.HttpURLConnection'](https://www.codacy.com/app/triplez/open-event-android/file/2406715500/issues/source?bid=3233850&fileBranchId=3233904#l12)

File: app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java [hunk 5]
```diff
@@ -2,18 +2,26 @@
...
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java]: Codacy Issue found: [Avoid unused imports such as 'java.net.URLConnection'](https://www.codacy.com/app/triplez/open-event-android/file/2406715500/issues/source?bid=3233850&fileBranchId=3233904#l15)

File: app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java [hunk 6]
```diff
@@ -2,18 +2,26 @@
 
 import android.content.Intent;
 import android.net.Uri;
+import android.util.Log;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java]: Codacy Issue found: [Avoid unused imports such as 'android.util.Log'](https://www.codacy.com/app/triplez/open-event-android/file/2406715500/issues/source?bid=3233850&fileBranchId=3233904#l5)

**Pull request closed without merging**